securedrop/source_app/utils.py

```diff
@@ -8,18 +8,25 @@
 from sqlalchemy.orm import sessionmaker
 from threading import Thread
 
+import typing
+
+if typing.TYPE_CHECKING:
```

Comment:
Remove the import of `Optional` from under the if statement, that should fix the lint error.

Reply:
So, remove lines 13-14?

Reply:
Don't we need to import `Optional` somewhere here?

Reply:
Sorry for the confusion, we have to keep the import statements, and maybe the best way is to mark `Optional` as strings in all of the usage lines, example is given below. 

```diff
diff --git a/securedrop/source_app/utils.py b/securedrop/source_app/utils.py
index e18766d00..fe01481b4 100644
--- a/securedrop/source_app/utils.py
+++ b/securedrop/source_app/utils.py
@@ -125,7 +125,7 @@ def normalize_timestamps(filesystem_id: str) -> None:
                 rc)
 
 
-def check_url_file(path: str, regexp: str) -> Optional[str]:
+def check_url_file(path: str, regexp: str) -> "Optional[str]":
     """
     Check that a file exists at the path given and contains a single line
     matching the regexp. Used for checking the source interface address
@@ -143,11 +143,11 @@ def check_url_file(path: str, regexp: str) -> Optional[str]:
         return None
 
 
-def get_sourcev2_url() -> Optional[str]:
+def get_sourcev2_url() -> "Optional[str]":
     return check_url_file("/var/lib/securedrop/source_v2_url",
                           r"^[a-z0-9]{16}\.onion$")
 
 
-def get_sourcev3_url() -> Optional[str]:
+def get_sourcev3_url() -> "Optional[str]":
     return check_url_file("/var/lib/securedrop/source_v3_url",
                           r"^[a-z0-9]{56}\.onion$")

```